tmt.spec

```diff
@@ -144,6 +144,17 @@ install -pm 644 %{name}/steps/provision/mrack/mrack* %{buildroot}/etc/%{name}/
 %files -n tmt+all -f %{_pyproject_ghost_distinfo}
 
 %changelog
+* Fri Apr 19 2024 Lukáš Zachar <lzachar@redhat.com> - 1.32.2
```

Comment:
With this we'll still have the rpmlint error for not having release in changelog entry. Ultimately it doesn't matter, as it's not a regression, but again just noting it.

The previous thread disappeared, sorry.

Reply:
So it either rpmlint complain or ignored %autorelease?

Reply:
I assume autorelease and autochangelog are kinda meant to be used together.
...maybe let's "fix" it in next release? :)

Reply:
ack., keeping as is for now.